symplyphysics/laws/waves/wavelength_from_velocity_and_frequency.py

```diff
@@ -0,0 +1,37 @@
+from sympy import pi
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+## The main characteristics of any wave are it's frequency, wavelength, magnitude and propagation speed.
+## Magnitude is power characteristic of wave.
+## Other characteristics are dependent from each over.
+
+## Definition: λ = V / (n * f)
```

Comment:
Something is wrong with this law. Refractive index does not matter unless there is wave passing from outer medium to refractive. I guess, V in this formula should be speed of light in vacuum.